web/templates/account/edit_security.html.eex

```diff
@@ -16,11 +16,11 @@
     <div class="pb4">
       <%= label f, "New security question", class: "f5 b db mb2" %>
       <%= select f, :security_question,
-        ["What were the last four digits of your childhood telephone number?": "What were the last four digits of your childhood telephone number?",
-        "What was the house number and street name you lived in as a child?": "What was the house number and street name you lived in as a child?",
-        "What time of the day were you born? (hh:mm)": "What time of the day were you born? (hh:mm)",
-        "What are the last five digits of your driver's licence number?": "What are the last five digits of your driver's licence number?",
-        "What is your grandmother's (on your mother's side) maiden name?": "What is your grandmother's (on your mother's side) maiden name?"
+        ["Favourite teacher's name?": "Favourite teacher's name?",
+        "First pet's name?": "First pet's name?",
+        "Town you were born in?": "Town you were born in?",
+        "Name of primary school?": "Name of primary school?",
+        "Name of your first boss?": "Name of your first boss?"
```

Comment:
I did end up doing this in a separate PR to have them finished & merged in and because it didn't really fit with the rest of the changes. I imagine I could've taken these out of here & put into a diff branch (?) but I didn't know how to do that.